docs/source/keywords.rst

```diff
@@ -151,6 +151,10 @@ General options
       * Aggressively vectorize loops
       * ``OFF``
 
+    * * ``Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE``
+      * Enables Kokkos behaving like a CMake language, see `Separate Compilation <building.html#separate-compilation-via-cmake-language>`_.
+      * ``OFF``
+
```

Comment:
```suggestion
    * * ``Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE``
      * Use native CMake language support
      * Analogous to CUDA/HIP as CMake language
      * ``OFF``
```

Reply:
This must be three lines not four for rendering the table correctly.